layers/meta-resin-genericx86/recipes-kernel/linux/files/realsense_hid_linux-yocto_4.4.patch

```diff
@@ -0,0 +1,181 @@
+From c8f0b4d94aca82dc7f8eae856db98a34941c3ce5 Mon Sep 17 00:00:00 2001
+From: Scott Ware <scott.r.ware@intel.com>
+Date: Mon, 27 Nov 2017 14:34:23 +0000
+Subject: [PATCH 2/4] Realsense HID
+
+Upstream-Status: Backpot
```

Comment:
needs also signed-ff by you, Theodor, in the correct order.
See https://github.com/resin-os/resin-intel/blob/master/layers/meta-resin-genericx86/recipes-kernel/linux/files/0001-Add-support-for-Quectel-EC20-modem.patch for example